pkg/parent/warn.go

```diff
@@ -0,0 +1,64 @@
+package parent
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
```

Comment:
is this repo using three groups (std, external, local) for imports? If not, (nit) remove this line

Reply:
yes, three groups: https://github.com/rootless-containers/rootlesskit/blob/da1dfffa441bad0cfeeb35e7ca6423e5f9ee215d/pkg/parent/parent.go#L4-L26